.github/workflows/build.yml

```
@@ -0,0 +1,39 @@
+    steps:
+      - id: skip_check
+        uses: fkirc/skip-duplicate-actions@v5
+        with:
+          cancel_others: 'false'
+          paths_ignore: '["**.md"]'
+
+      - uses: actions/checkout@v4
+        if: steps.skip_check.outputs.should_skip != 'true'
+
+      - name: Build Dependencies
+        if: steps.skip_check.output.should_skip != 'true'
```

actionlint (1.7.4)

property "output" is not defined in object type {conclusion: string; outcome: string; outputs: {string => string}}

(expression)